inc/ThirdParty/Plugins/Slider/LayerSlider.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Did you test if we still need this compatibiliy file? Seeing how old the original code is, a lot can have happened since

Reply:
@Tabrisrp Yes I tried that locally and I can see that layerslider alreay uses img tag with dimensions already set as in the following screenshot.

![image](https://user-images.githubusercontent.com/15707971/100135427-499c1200-2e92-11eb-90ea-4bcce031c537.png)

So our code won't affect it at all, even our code affected it and added dimensions, this won't make a conflict with layerslider.

@wp-media/php what do u think?

Reply:
Ok, so we don't need the compatibility file, right?

Reply:
@Tabrisrp Yes I think so.

Reply:
great, so let's just remove it along the old compatibility file 👍🏼

Reply:
Ok will so that now.